pkgs/data/fonts/apple-color-emoji/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, pkgs, ... }:
+
+stdenv.mkDerivation rec {
+  name = "apple-color-emoji";
+  version = "2020-04-13";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "samuelngs";
+    repo = "apple-emoji-linux";
+    rev = "8c247a77eb0abe9fca1011a6ab75520dc761ff37";
+    sha256 = "1r0laf5g4gjrrv2xzr1l5bvkn7px9k02p6wknxaafm3mrmpxrk23";
+  };
+
+  nativeBuildInputs = with pkgs; [ which imagemagick optipng zopfli pngquant ]
+  ++ (with python3Packages; [ python fonttools nototools ]);
+
+  enableParallelBuilding = true;
+
+  installPhase = ''
+    install -m444 -Dt $out/share/fonts/apple-color-emoji AppleColorEmoji.ttf 
```

The Makefile has an install method you can reuse, just need to patch the install path.
```suggestion
    substituteInPlace Makefile --replace 'HOME/.local/' 'out/'
```